config/config.go

```diff
@@ -161,6 +166,174 @@ type AuthConfig struct {
 	AppendMetadataHeaders bool          `yaml:"append_metadata_headers,omitempty" json:"append_metadata_headers,omitempty"`
 }
 
+// EnvConfigs
+type EnvConfigs struct {
+	// A list of files containing environment config
+	ConfigFiles []string `yaml:"config_files,omitempty" json:"config_files,omitempty"`
```

Comment:
Do we want to actually prescribe that these must be "files", or should we be more generic and support URIs, where files can be referencing using the standard `file://` scheme. So in this case we might call this `ConfigURIs` or `ConfigRefs`

Reply:
Changed to `ConfigURIs` and added a comment that only local file system is supported currently.